src/wallet/asyncrpcoperation_sendmany.h

```diff
@@ -41,6 +42,12 @@ struct AsyncJoinSplitInfo
     CAmount vpub_new = 0;
 };
 
+// A struct to help us track the witnesses and anchor for a given JSOutPoint
+struct WitnessAnchorData {
+	std::vector<boost::optional<ZCIncrementalWitness>> witnesses;
```

Comment:
See my comment above; I don't see anywhere in this PR where `witnesses` would ever hold more than one value.